cmd/soroban-cli/src/signer/secure_store.rs

```diff
@@ -0,0 +1,59 @@
+use sep5::SeedPhrase;
```

Comment:
i'm not sure if this needs its own mod, or exactly where to put this, but I wanted to pull the code out where both `add` and `generate` could use it